src/MIDebugEngine/Engine.Impl/EngineUtils.cs

```diff
@@ -52,7 +52,8 @@ internal static async Task<string> GetAddressDescriptionAsync(DebuggedProcess pr
             if (location == null)
             {
                 string addrFormat = proc.Is64BitArch ? "x16" : "x8";
-                location = ip.ToString(addrFormat, CultureInfo.InvariantCulture);
+                // Make sure its prefaced by 0x
+                location = String.Format(CultureInfo.InvariantCulture, "0x{0}", ip.ToString(addrFormat, CultureInfo.InvariantCulture));
```

Comment:
>"0x [](start = 71, length = 3)

Are you sure you want this? Normally in the VS debugger we don't `0x` pointers in contexts where it is obvious that something is a pointer (ex: memory window, disassembly window, `foo.dll!00112233` in the callstack, etc)

Reply:
Ok. I can change that . That makes sense